examples/vision/mobilevit.py

```diff
@@ -30,17 +30,16 @@
 ## Imports
 """
 
+import os
 import tensorflow as tf
 
-from keras.src.applications import imagenet_utils
+os.environ["KERAS_BACKEND"] = "tensorflow"
 
-# For versions <TF2.13 change the above import to:
-# from keras.applications import imagenet_utils
-from tensorflow.keras import layers
-from tensorflow import keras
+import keras
+from keras import layers
+from keras.src.applications import imagenet_utils
```

Comment:
This is a private API, we should not rely on it. You could reimplement the `correct_pad` utility in the example, or just get rid of it entirely.